Libplanet.Store/DefaultStore.cs

```diff
@@ -105,7 +104,7 @@ public class DefaultStore : BaseStore
         private readonly LruCache<TxId, object> _txCache;
         private readonly LruCache<BlockHash, BlockDigest> _blockCache;
 
-        private readonly MemoryStream _memoryStream;
+        private readonly MemoryStream? _memoryStream;
```

Comment:
Looks like we can get rid of this altogether.

Reply:
How about this code below?
```csharp
_db = new LiteDatabase(new MemoryStream(), disposeStream: true);
```